.github/actions/test-setup/action.yaml

```diff
@@ -15,7 +15,15 @@ runs:
         path: ~/.local/share/pnpm/store/v4
         key: ${{ runner.os }}-node-${{ inputs.node-version }}-pnpm-${{ inputs.pnpm-version }}-lock-${{ hashFiles('**/pnpm-lock.yaml') }}
         restore-keys: |
-          ${{ runner.os }}-pnpm
+          ${{ runner.os }}-node-
```

Comment:
This `restore-key` would have been incorrect, had resolved here but can move if out of scope